core/math/quat.cpp

```diff
@@ -248,3 +248,51 @@ void Quat::set_axis_angle(const Vector3 &axis, const real_t &angle) {
 				cos_angle);
 	}
 }
+
+// Decomposes a Quaterian into a swing and twist, where twist is the rotation around the
```

Comment:
Typo. Quaternion.

I agree that this should be in the quaternion class and not as a subclass.